handlers/CaseStudies/components/CaseStudiesContent/styles.css

```diff
@@ -57,27 +59,19 @@
   position: absolute;
   top: 50%;
   left: 50%;
-  opacity: 0.5;
+  opacity: 1;
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
   max-width: 300%;
   min-width: 100%;
-  transition: opacity 0.3333s ease-in;
+  /*transition: opacity 0.3333s ease-in;*/
 }
 
 @media screen and (min-width: 601px) {
   .CaseStudiesContent-item {
     display: inline-block;
-    height: 14em;
+    height: calc(60vw * 0.5);
```

Comment:
`vw` units have pretty whacky browser support (namely Safari). Let’s make sure this works for them in QA.